File: server/order-service/src/main/java/com/sudo248/invoiceservice/controller/dto/UserDto.java
package com.sudo248.invoiceservice.controller.dto;

import com.sudo248.invoiceservice.repository.entity.Gender;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserDto {
    private String userId;

    private String fullName;

    private String phone;

    private Date dob;

    private String bio;

    private String avatar;

    private String cover;

    private AddressDto address;

    private Gender gender;
}

File: server/auth-service/src/main/kotlin/com/sudoo/authservice/controller/dto/UserDto.kt
package com.sudoo.authservice.controller.dto

import com.sudoo.authservice.model.Gender
import java.time.LocalDate

data class UserDto(
    val userId: String,
    val fullName: String,
    val emailOrPhoneNumber: String,
    val dob: LocalDate = LocalDate.now(),
    val bio: String = "New user",
    val avatar: String = "user_default.png",
    val cover: String = "user_default.png",
    val address: AddressDto = AddressDto(),
    val gender: Gender = Gender.OTHER,
)